ansible/roles/logstash/templates/pipeline.conf

```diff
@@ -0,0 +1,30 @@
+input {
+    beats { 
+        port => 5044
+    }
+    file {
+        path= "/media/ephemeral0/logs/accumulo/*.log"
+    }
+}
+filter {
+  if [fileset][module] == "logstash" {   
+    grok {
+        match => { "message" => "%{TIMESTAMP_ISO8601:TIMESTAMP} \[%{GREEDYDATA:module}\.%{GREEDYDATA:class_name}\] %{WORD:level}: %{GREEDYDATA:message}\[%{GREEDYDATA:module}\]"}
```

Comment:
For maintainability purposes, please put an example of the line being parsed in a comment before each match statement.  That will make this a lot easier to understand.